src/TiciTacaToeyGameEngine.ts

```diff
@@ -229,17 +250,18 @@ class TiciTacaToeyGameEngine implements GameEngine {
         break;
       }
       case MessageTypes.JOIN_GAME: {
+        const gameId = message.gameId;
         if (!(message.playerId in this.players)) {
           this.players = addPlayer(
             this.players,
             message.playerId,
             "",
             message.connection
           );
+          this.games[gameId].timers[message.playerId] = new Timer(
```

Comment:
Do not directly mutate this object, perform this operation on line 277 below where the game object is being updated